### PR TITLE
Update SR830_ext for use with SCFG

### PR DESCRIPTION
**Pull request by ThorvaldLarsen**
Update SR830_ext for use with the new stationconfigurator. Also add resistance and conductance measurements (this is a complete copy of the SR86x_ext only on a different subclass).

Still need to test it.

### CODE REVIEW

**Comment by ThorvaldLarsen** [comment]
This has now been tested and been shown to work. I did not test the conductance channel part but this has not really been changed from what it was before.

Walked through lab and nobody is using this driver (it is also old and not compatible with the station configurator) so should be fine to modify it. 
Everyone is instead using manual paramters in the init script to get conductance and resistance. This was done on at least 5 different setups so there is a real need to have this in the driver.

File: qdev_wrappers/customised_instruments/SR830_ext.py [hunk 1]
```diff
@@ -37,81 +34,73 @@ def get(self):
...
-
-        self.add_parameter('resistance',
-                           label='{} Resistance'.format(self.name),
+       	self.add_parameter(name='resistance',
+                           label='Resistance',
+                           unit='Ohm',
                            get_cmd=self._get_resistance,
+                           get_parser=float)
+        
+        self.add_parameter(name='g_X',
```
Comment (jenshnielsen) [inline qdev_wrappers/customised_instruments/SR830_ext.py]: This indentation seems to be off

Comment (jenshnielsen) [reply qdev_wrappers/customised_instruments/SR830_ext.py]: Or rather above, I think some taps have ended up in the code

Comment (ThorvaldLarsen) [reply qdev_wrappers/customised_instruments/SR830_ext.py]: Weird. It still runs though. Tried redoing the indentation but feel like there is an automatic way of doing that which I haven't learned yet...

Comment (jenshnielsen) [reply qdev_wrappers/customised_instruments/SR830_ext.py]: Depending on the editor there is usually a convert taps to space setting and a indentation config

**Comment by Dominik-Vogel** [comment]
@ThorvaldLarsen you added a factor 1/2 for the conductance. Is this not a problem for everybody to suddenly have different results? (I mean have you communicated this? we should somehow capture this in meta data...)

**Comment by ThorvaldLarsen** [comment]
It is already in the metadata with the unit being `e^2/h` instead of `2e^2/h`, so will be obvious. The choice of unit was based what people are using at the moment.

Also, nobody was using the old driver so it will not be a surprise change for anyone. Instead everyone have manual parameters in the init script using the bare qcodes driver. So changing this will not change any code for anyone but rather give a better option than what is currently being done.

**Comment by Dominik-Vogel** [comment]
ok, I just overlooked that you also changed the unit...

**Pull request merged**